index.html

```diff
@@ -15,6 +15,17 @@
 <body>
   <header class="main-head">
```

Comment:
- [ ] OPTIONAL: The whole document has an overflow, I have to manually scroll the document left and right to see the content, I strongly recommend addressing this issue and making the page responsive